opencensus/metrics/export/cumulative.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import six
+
+from opencensus.metrics.export import metric_descriptor
+from opencensus.metrics.export import gauge
+
+
+class CumulativePointLong(gauge.GaugePointLong):
+    """A `GaugePointLong` that cannot decrease."""
+
+    def set(self, val):
```

Comment:
We don't need a `set` method for Cumulative. See https://github.com/census-instrumentation/opencensus-java/pull/1853.

Reply:
21ad4b2 leaves the class hierarchy in place, but means `DerivedGaugePoint` now calls the hidden `_set` method. Another option is to flatten these classes out -- have separate classes for derived gauges and cumulatives and duplicate some code.

Reply:
https://github.com/census-instrumentation/opencensus-python/commit/21ad4b23db9654d9dfeb9e3099d9fce72f89781e looks good to me.